default.nix

```diff
@@ -1 +1 @@
-with import <nixpkgs> {}; stdenv.mkDerivation { name = "malloy"; buildInputs = [ nodejs-16_x jdk8 google-cloud-sdk postgresql git cacert openssh jq fakeroot]; }
+with import <nixpkgs> {}; stdenv.mkDerivation { name = "malloy"; buildInputs = [ nodejs_18 jdk8 google-cloud-sdk postgresql git cacert openssh jq fakeroot]; }
```

Comment:
Does this need to be `nodejs-18_x`? nix isn't liking `nodejs_18`

Reply:
**tl;dr**: that's exactly right; I have authored a commit that references `nodejs-18_x` instead.

---

Thanks for proxying this information from CI.  Although both `nodejs-18_x` and `nodejs_18` are both present in the latest releases of [`nixpkgs`](https://github.com/NixOS/nixpkgs) -- and therefore the latest docker image, such as the latest release [`2.15.2`](https://hub.docker.com/r/nixos/nix/tags) -- `gcr.io` appears to have a cached version of `nixos/nix`, which is older.

For more information about Cloud Build's caching semantics, see [cloud.google.com:Interacting with Dockerhub Images](https://cloud.google.com/build/docs/interacting-with-dockerhub-images).

In a separate PR https://github.com/malloydata/malloy/pull/1317 is an experiment to see if we can merely instruct Cloud Build to use the most recent pinned `nixos/nix` docker image.